readme/working-with-bns.md

```diff
@@ -0,0 +1,98 @@
+# Introduction
+
+When working with the Monaco Protocol, one thing that you will notice is that fields you expect to be numbers (stake, timestamp) - aren't.
+
+Instead, these numbers are represented as a `BNs`, or big numbers. These `BNs` are excellent for brevity but not for readability as they need to be converted back into a `number`.
```

Comment:
It's maybe worth mentioning that BN's aren't a monaco thing but a solana thing, and they are used because javascript `number` will not be able to hold the full range of numbers supported by a u64.

Reply:
yeah good point, was thinking about this then forgot about this 🙇